prom/CMakeLists.txt

```diff
@@ -137,4 +137,4 @@ set(CPACK_PACKAGE_HOMEPAGE_URL https://github.internal.digitalocean.com/timeseri
 
 include(CPack)
 install(TARGETS prom ARCHIVE)
-install(DIRECTORY include/ DESTINATION /usr/include/)
+install(DIRECTORY include/ DESTINATION include)
```

Comment:
It is still better to use [`GNUInstallDirs`](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) module so that distros can choose the installation location (e.g. using multiple outputs in Nixpkgs).